src/flint/helpers.py

```diff
@@ -46,24 +56,17 @@ def log_telemetry(
 
 
 def download_audio_message(audio_url, user_id):
-    # Get the url of the voice message file
-    url = requests.get(audio_url).url
-
-    # Send a HEAD request to the audio URL to get the size of the audio packet
-    head = requests.head(url)
-    # Get the content type of the voice message file
-    content_length = head.headers.get("Content-Length")
-    logger.info(f"Audio message content length is {content_length}")
-
-    # If the audio message is larger than 10 MB, return None
-    if int(content_length) > 10 * 1024 * 1024:
-        logger.info(f"Audio message is larger than 10 MB, skipping")
-        raise ValueError(f"Audio message is larger than 10 MB")
+    headers = {
+        "Authorization": f"Bearer {whatsapp_token}",
+    }
+    response = requests.get(audio_url, headers=headers)
 
     # Create output file path with user_id and current timestamp
     filepath = f"/tmp/{user_id}_audio_{int(time.time() * 1000)}.ogg"
     # Download the voice message OGG file
-    urllib.request.urlretrieve(url, filepath)
```

Comment:
Changed because there isn't a neat way to keep the `urlretrieve` method with headers.